mysite/users/views.py

```diff
@@ -56,4 +56,16 @@ def profileUpdate(request, pk):
             edit.save()
             return redirect('profile')
     form = UserProfileForm()
-    return render(request,'user/profile.html',{'form': form})
\ No newline at end of file
+    return render(request,'user/profile.html',{'form': form})
+
+@login_required
+def editProfile(request):
+    if request.method == 'POST':
+        # server = get_object_or_404(Profile, pk=pk)
```

Comment:
Please remove the commented lines unless it's some info regarding the functionlaity.